### ISSUE TITLE
Context menu cannot be opened on some webpages

### ISSUE THREAD

**Issue by pylorak** [pylorak/reloadmatic#3]
For an affected webpage, see a Google Spreadsheet document.

Needs [FF bug #1414633](https://bugzilla.mozilla.org/show_bug.cgi?id=1414633) to be solved first. Hence this is a WebExtension API limitation.

**Comment by pylorak** [comment]
Tracking new ticket https://bugzilla.mozilla.org/show_bug.cgi?id=1416030 .

**Comment by pylorak** [comment]
Depending on ticket #9 .

**Comment by pylorak** [comment]
Now that tab header context menus are possible, this is a non-issue anymore. Important: You need both ReloadMatic 1.7.0 (already published) AND Firefox 60 (not yet released) for this feature to work.
